insights/specs/sos_archive.py

```diff
@@ -98,7 +98,11 @@ class SosSpecs(Specs):
         simple_file("sos_commands/foreman/foreman-debug/qpid_stat_subscriptions"),
         simple_file("sos_commands/foreman/foreman-debug/qpid-stat-u")
     ])
-    rabbitmq_report = simple_file("sos_commands/rabbitmq/rabbitmqctl_report")
+    rabbitmq_report = first_of([
```

Comment:
Since the `glob_file` means a couple of files which matches the wildcard string will get this parser being fired, this update will change the structure of the parser result, from a single Parser object to a list of Parser objects, which will break all the existing rules depend on this Parser.

If there any difference (e.g. the usage in rules) between this Parser against containers or against normal-hosts, but to avoid breaking the existing rules, seems we'd better add a new spec&parser for the new one against containers.

Or, is it possible to add a new command to the `sos` package to collect a single `rabbitmqctl_report` for one container? Just like: https://github.com/RedHatInsights/insights-core/issues/1976

Reply:
Hi @xiangce The * is there because the file changes per server, it can be any number. Do you have anything to catch that in insights?

Reply:
@zerodayz - per the discussion, I have opened an issue #2049 for this PR. And I will submit another PR for the parser and close this one. Thanks.